### ISSUE TITLE
RFC: What is OrbitDB?

### ISSUE THREAD

**Issue by RichardLitt** [simpleaswater/rfcs#24]
**[REQUIRED] What new topic/thread/tutorial are you proposing? Please describe.**
The current https://simpleaswater.com/ has some great guides on What is *. I think that [OrbitDB](https://github.com/orbitdb/orbit-db) would also benefit from one of these. It might be simpler to do than #16, as well.

**Comment by vasa-develop** [comment]
@RichardLitt That's an awesome idea!

Do you have any references that we could use for the tut(s)? 
Plus could you share your email here? We would add you as a contributor on simpleaswater, so that we could collaborate easily on this.

**Comment by RichardLitt** [comment]
Yes! Take a look at the [Field Guide](https://github.com/orbitdb/field-manual) and at the [Web3 Workshop](https://github.com/orbitdb/web3-workshop). 

Unsurprisingly, you'll find me at richardlitt@orbitdb.org. :)

**Comment by vasa-develop** [comment]
@RichardLitt Sent you the invite. And please do check the spam :)

Thanks for your contribution!

**Comment by vasa-develop** [comment]
@0zAND1z @RichardLitt 
I had some problems with my server, so I had to send new email invites to both of you guys.
Please accept the invites as per your convenience.

Thanks :)

**Comment by RichardLitt** [comment]
No problem! I joined again; but I'm still not quite sure what to do, after joining.

**Comment by vasa-develop** [comment]
@RichardLitt Thanks for joining again. We will draft an article this week on OrbitDB. I will send you the link, and you can let me vet it a bit :)
